docker-compose.test.yml

```
@@ -1,6 +1,6 @@
 # Docker Compose Configuration for Test Environment
 
-version: "3.9"
+version: "3.10"
```

This is a property of the compose file for backward compatibility.

```suggestion
version: "3.9"
```